proteuslib/tools/parameter_sweep.py

```diff
@@ -129,9 +246,38 @@ def _default_optimize(model, options=None, tee=False):
 
 # ================================================================
 
+def _process_sweep_params(sweep_params):
+
+    sampling_type = None
+
+    # Check the list of parameters to make sure they are valid
+    for i, key in enumerate(sweep_params.keys()):
+
+        # Convert to using Sample class
+        if isinstance(sweep_params[key], (list, tuple)):
+            sweep_params[key] = LinearSample(*sweep_params[key])
+
+        # Get the type of sampling
+        if isinstance(sweep_params[key],FixedSample):
+            current_sampling_type = "fixed"
```

Comment:
We could set `current_sample_type` to `FixedSample` as opposed to a string.

Reply:
added an Enum class